extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -10,6 +11,7 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 ], function (
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Function has 8 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)